doc/tutorials/02-charged_system/02-charged_system-1.ipynb

```diff
@@ -4,33 +4,52 @@
    "cell_type": "markdown",
```

Comment:
<p>please use ALL_CAPITAL letters for constants</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3914/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>